src/use-url-params.ts

```diff
@@ -17,7 +17,7 @@ import { UrlMatchResults } from './types';
  *
  * @returns object with a `matches` key, and (if it matched) path and query parameters
  */
-export default (pattern: string): UrlMatchResults => {
+export default (pattern?: string): UrlMatchResults => {
```

Comment:
this should have always been optional! This better matches the interface of the underlying library.